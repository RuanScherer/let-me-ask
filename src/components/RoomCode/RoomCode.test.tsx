import { render, screen } from '@testing-library/react';
import { RoomCode } from './';

describe('Room Code Component', () => {
	test('Show room code', () => {
		const code = '-Ab-1jH-';
		render(<RoomCode roomCode={code} />);

		expect(screen.getByText(/-Ab-1jH-/)).toBeInTheDocument();
	});

	test('Snapshot must always be the same', () => {
		const { container } = render(<RoomCode roomCode="-Ab-1jH-" />);

		expect(container.firstChild).toMatchSnapshot();
	});
});

export { };
