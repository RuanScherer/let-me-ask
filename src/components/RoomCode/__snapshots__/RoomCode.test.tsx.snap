// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Room Code Component Snapshot must always be the same 1`] = `
<button
  class="room-code"
>
  <div>
    <img
      alt=""
      src="copy.svg"
    />
  </div>
  <span>
    Sala #
    -Ab-1jH-
  </span>
</button>
`;
